webcompat/static/js/lib/flashed-messages.js

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This file is only included in the page when the Flask view is rendered with
+// a flash message. It's responsible for triggering that message as defined
+// in flash-message.js. See layout.html for how category and message data
+// attributes are set.
+
+var currentScript = document.currentScript;
```

Comment:
Hm, this is tricky because we load this script at the bottom of the page to be sure `flash-message.js` is already loaded -- which comes just before `</body>`. So there's an ordering dependency. Also, we can't just set these attributes on `<body>` or `<main>`, then load the script down below because calling `get_flashed_messages` clears them from the session: https://github.com/pallets/flask/blob/a63a4cb034955f5d9c4b14f87a5337d94ba472a1/flask/helpers.py#L425.

There's a polyfill, but it only works for IE6 - 10, and *not* IE11 (which is what ships in Windows 10). Since Edge works with this approach, and Microsoft wants their users on Edge... I'm inclined to just forget about old IE support anyways.... :/